### PR TITLE
wasm-interp: Fix catch handlers correctly

### PR DESCRIPTION
**Pull request by SoniEx2**
oops :sweat_smile: (local decl count != local count)

### CODE REVIEW

**Comment by SoniEx2** [comment]
@sbc100

File: test/regress/interp-try-catch-multi-locals.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+;;; TOOL: run-interp-spec
+;;; ARGS*: --enable-exceptions
+;;; NOTE: ref: issue-2476
+(module
+  (tag $e0)
+  (func (export "test") (result i32)
+    (local $a i32)
+    (local $b i32)
```
Comment (sbc100) [inline test/regress/interp-try-catch-multi-locals.txt:8]: Can we just update an existing test instead maybe ?

**Review by sbc100 — APPROVED** [review]

File: test/regress/interp-try-catch-multi-locals.txt (hunk 1 above)
Comment (SoniEx2) [reply test/regress/interp-try-catch-multi-locals.txt]: like so?

**Review by sbc100 — APPROVED** [review]
Looks reasonable to me

**Pull request merged**